test/parallel/xyk-pallet.API.fees.tripleTokens.test.ts

```diff
@@ -1,6 +1,7 @@
 /*
  *
  * @group xyk
+ * @group market
```

Comment:
only this test with this group? or u are using it for debugging?

Reply:
I change all xyk tags to market in files where I delete xyk link